cmd/fs-v1-helpers.go

```diff
@@ -161,11 +157,8 @@ func fsStat(ctx context.Context, statLoc string) (os.FileInfo, error) {
 		logger.LogIf(ctx, err)
 		return nil, err
 	}
-	fi, err := os.Stat((statLoc))
+	fi, err := os.Stat(statLoc)
 	if err != nil {
-		if err != errFileNotFound {
-			logger.LogIf(ctx, err)
-		}
```

Comment:
we can still log this like:
`if os.IsNotExist(err) {log()}`

current code problem is that it uses errFileNotFound causing unnecessary logs

Reply:
We need to be careful about that @krishnasrinivas just doing `os.IsNotExist(err)` won't work since fsStat() is actually a call used by two other functions as well. That is why I moved the code outside such that the caller can ignore it as needed.

Reply:
if we conside fsStat() as a low level call, then the caller fsStatDir() fsStatFile() is not logging/ignoring

```
// Lookup if directory exists, returns directory attributes upon success.
func fsStatDir(ctx context.Context, statDir string) (os.FileInfo, error) {
	fi, err := fsStat(ctx, statDir)
	if err != nil {
		return nil, osErrToFSFileErr(err)
	}
	if !fi.IsDir() {
		logger.LogIf(ctx, errFileAccessDenied)
		return nil, errFileAccessDenied
	}
	return fi, nil
}

// Lookup if file exists, returns file attributes upon success.
func fsStatFile(ctx context.Context, statFile string) (os.FileInfo, error) {
	fi, err := fsStat(ctx, statFile)
	if err != nil {
		return nil, osErrToFSFileErr(err)
	}
	if fi.IsDir() {
		logger.LogIf(ctx, errFileAccessDenied)
		return nil, errFileAccessDenied
	}
	return fi, nil
}
```